controller/deleteUser.js

```diff
@@ -0,0 +1,22 @@
+const { default: async } = require("async");
```

Comment:
No need to make a new file for every controller. You just need to create a userController file and write all the controllers related to user routes there.